platform/ios/src/MGLMapView.mm

```diff
@@ -450,7 +450,9 @@ - (void)commonInit
     _pendingLongitude = NAN;
     _targetCoordinate = kCLLocationCoordinate2DInvalid;
 
-    [MGLMapboxEvents pushEvent:MGLEventTypeMapLoad withAttributes:@{}];
+    if ([UIApplication sharedApplication].applicationState != UIApplicationStateBackground) {
+        [MGLMapboxEvents pushEvent:MGLEventTypeMapLoad withAttributes:@{}];
+    }
 }
```

Comment:
@1ec5 I kept this in commonInit since, when map view is first initialized it cannot have started to observe life cycle events and therefore will never call `wakeGL:` the first time it is loaded. I did however add the guard so that the 
turnstile event will not be attempted if this happens in the background.